docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md

```diff
@@ -27,8 +27,17 @@ This tutorial shows how to provision Kyma Runtime on Azure using Kyma Environmen
    ```bash
    export AUTHORIZATION_HEADER="Authorization: Bearer $ACCESS_TOKEN"
    ```
-
-3. Make a call to the Kyma Environment Broker to create a Runtime on Azure. Find the list of possible request parameters [here](03-01-service-description.md).
+Alternatively, you can perform `kubectl port-forward` on the chosen pod to expose it on your local machine. Expose it to port `8080`:
+```bash
+  kubectl port-forward -n kcp-system deployments/kcp-kyma-environment-broker 8080
+```
+You should get information about the successful forwarding. For example:  
```

Comment:
```suggestion
As a result, you get information about the successful forwarding. For example:  
```

Try not to use `should`, as it leaves too much room for ambiguity: `should`, but there's an option you get something else? What then? 

Also, I wouldn't say this is necessary to include this result here at all.

Reply:
Tbh I practically copied it from another doc, but - sure - there's always scope for improvement :)